apps/web/playwright/bookings-list.e2e.ts

```diff
@@ -61,6 +68,155 @@ test.describe("Bookings", () => {
         secondUpcomingBooking.locator(`text=${bookingWhereFirstUserIsOrganizer!.title}`)
       ).toBeVisible();
     });
+    test("collective eventType booking should be visible to team admin", async ({
+      page,
+      users,
+      bookings,
+    }) => {
+      const { owner1, owner2, commonUser, team1_teammate1 } = await createTeams(users);
+
+      const { team: team1 } = await owner1.getFirstTeamMembership();
+      const scenario = {
+        schedulingType: SchedulingType.COLLECTIVE,
+        teamEventTitle: `collective-team-event`,
+        teamEventSlug: slugify(`collective-team-event-${randomString(5)}`),
+      };
+
+      const eventType = await createTeamEventType(owner1, team1, scenario);
+      const { id: eventId, slug: evnetSlug } = eventType;
+
+      await prisma.host.createMany({
+        data: [
+          {
+            userId: commonUser.id,
+            eventTypeId: eventId,
+            isFixed: true,
+          },
+          {
+            userId: team1_teammate1.id,
+            eventTypeId: eventId,
+            isFixed: true,
+          },
+        ],
+      });
+
+      // to test if booking is visible to team admin/owner even if he is not part of the booking
+      await prisma.host.deleteMany({
+        where: {
+          userId: owner1.id,
+          eventTypeId: eventId,
+        },
+      });
+
+      await bookEvent({ pageFixture: page, eventType, team: team1 });
+
+      // booking should be visible for the team host even though he is not part of the booking
+      await bookingVisibleFor({ user: owner1, pageFixture: page, eventType, shouldBeVisible: true });
+      // booking should not be visible for other team host
+      await bookingVisibleFor({ user: owner2, pageFixture: page, eventType, shouldBeVisible: false });
+      // booking should be visible for commonUser as he is part of the booking
+      await bookingVisibleFor({ user: commonUser, pageFixture: page, eventType, shouldBeVisible: true });
+      // booking should be visible for team1_teammate1 as he is part of the booking
+      await bookingVisibleFor({ user: team1_teammate1, pageFixture: page, eventType, shouldBeVisible: true });
```

Comment:
This function's name currently reads like it would return a boolean that you would then assert against. But it currently does the assertion within the function. In these cases, it's better to put `assert` in the function so it's more clear. `assertBookingVisibleFor`